### ISSUE TITLE
Issue with Colombian numbers

### ISSUE THREAD

**Issue by shanafme** [catamphetamine/libphonenumber-js#108]
Hello.  The Colombian phone number 5713144019538 seems to validate to a valid number (using "isValidNumber") when using libphonenumber-js.  However, when using Google's libPhoneNumber, this number is marked as being invalid.  

**lipbhonenumber-js result (valid):**
![image](https://user-images.githubusercontent.com/765569/29425740-5cd0f99c-8352-11e7-87b7-42bf58cf0622.png)

**Google libPhoneNumber result (not valid):**
![image](https://user-images.githubusercontent.com/765569/29425845-b4ac07ba-8352-11e7-98ec-c6cb1c1ec9fb.png)

Any ideas?  Thank you for this very useful tool.

**Comment by catamphetamine** [comment]
Now read this once again:
https://github.com/catamphetamine/libphonenumber-js#bug-reporting

Perhaps I should clarify that a bit more: the demo doesn't use extended validation metadata.
I'm gonna update the README to reflect that.

**Comment by catamphetamine** [comment]
I the bug reporting section:

> Phone number validation bugs should **only** be reported if they appear when using custom metadata functions fed with `metadata.full.json` because by default all functions in this library use the reduced metadata set which results in looser validation than the original Google `libphonenumber`'s. The [demo page](https://catamphetamine.github.io/libphonenumber-js/) also uses the reduced metadata set and therefore its validation is also looser than the original Google `libphonenumber`'s. That being said, it's very unlikely that any phone number validation inconsistencies could even exist. This paragraph is here just to prevent people from reporting "validation inconsistency" issues.

**Comment by shanafme** [comment]
Thank you for the response, and I apologize for opening an unfounded issue.  

One question though - what is the correct NPM command/procedure to build the libPhoneNumber-js bundle with the full metadata set?  I was able to get around this by downloading the project and then replacing the two "minimal" metadata sets with the text contained in the "full" metadata set.  I then ran 
`npm install`
`npm run browser-build`
to build the bundle.  I'm sure there was a better way, I'm just not familar enough with Node to have figured it out.
